src/content/overview/observability/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Observability
+description: Summary of the observability capabilities encompassed within the Giant Swarm platform.
```

Comment:
```suggestion
description: Everything about our observability capabilities that give you visbility and insights into the Giant Swarm platform and your developers workloads.
```

Reply:
Fill words, and similar repetitive stuff like visibility≈insights.

```suggestion
description: Observability tooling to provide you with visibility into the Giant Swarm platform, your cluster fleet and application workloads.
```